Enterprise-CMCS/eAPD
# Dmirano/4551 ffp screen reader bug

Resolves #4551, #4405 

### Description
Resolves screen reader bug reading out each choice last. The problem was because of the aria components exclusive to the cms design system's ChoiceList component. The solution was to create our own ChoiceList wrapper, and removing these aria tags from the components. Also replaced everywhere we use the ChoiseList component, which resolves #4405 as well.

### Chromatic Link

### Significant changes or possible side effects

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ChoiceList component | selected or not selected | component renders correctly | jest |
| ChoiceList component | component is rendered | component does not contain aria-live tags | jest |

### Steps to manually verify the bug fix (windows only)

1. Navigate to the Budget and FFP page, ffp split field
2. Open up a screen reader (can't be the apple screen reader since the bug doesn't happen) 
3. Verify the choices get read out loud first

### Steps to manually verify the ChoiceList changes

1. Verify ChoiceList components work as before
2. ChoiceList fields affected:
  - Update Type fields (MMIS)
  -  Medicaid Business Areas (MMIS)
  - Budget and FFP page: ffp split field (MMIS)
  - Assurances and Compliance
 3. Also verify ChoiceList story is correct

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
